manual/source/examples/h5py/simple_example_write1/index.rst

```diff
@@ -1,7 +1,7 @@
-.. _Example-writer_1_3:
+.. _Example-simple_example_write1:
 
-``h5py`` example writing the simplest NeXus data file
-#####################################################
+Write a NeXus HDF5 file
+#######################
 
 In this example, the 1-D scan data will be written into the simplest
```

Comment:
Been thinking lately that a simpler data file is possible, with no additional attributes.  (So, should the wording here be changed?) But this may be the simplest *useful* data file.

This is additional content: https://github.com/nexusformat/definitions/blob/47f61e432f992b7793bfcec39e6f9f7bea9948c0/manual/source/examples/h5py/simple_example_test/simple_example_test_write.py#L57-L64

https://github.com/nexusformat/definitions/blob/47f61e432f992b7793bfcec39e6f9f7bea9948c0/manual/source/examples/h5py/simple_example_test/simple_example_test_write.py#L68

https://github.com/nexusformat/definitions/blob/47f61e432f992b7793bfcec39e6f9f7bea9948c0/manual/source/examples/h5py/simple_example_test/simple_example_test_write.py#L73-L74

https://github.com/nexusformat/definitions/blob/47f61e432f992b7793bfcec39e6f9f7bea9948c0/manual/source/examples/h5py/simple_example_test/simple_example_test_write.py#L77

https://github.com/nexusformat/definitions/blob/47f61e432f992b7793bfcec39e6f9f7bea9948c0/manual/source/examples/h5py/simple_example_test/simple_example_test_write.py#L80

https://github.com/nexusformat/definitions/blob/47f61e432f992b7793bfcec39e6f9f7bea9948c0/manual/source/examples/h5py/simple_example_test/simple_example_test_write.py#L82-L89

Reply:
The test scripts aren't actually used in the docs.